docs/core/install/localized-intellisense.md

```diff
@@ -107,9 +106,9 @@ Once you've installed the desired language packs, modify your Visual Studio sett
 
 1. On the start window, choose **Continue without code**.
 
-1. On the main menu, select **Tools** > **Options**. The Options dialog opens.
+1. On the menu bar, select **Tools** > **Options**. The Options dialog opens.
```

Comment:
I think the correct name is "menu bar" not "main menu".

Reply:
@gewarren do you know? I think I saw this in the VS docs if I'm not mistaken.

Reply:
Menu bar sounds more correct to me. It's not specific to VS though.